web/src/beta/lib/core/engines/Cesium/Feature/Marker/index.tsx

```diff
@@ -28,7 +28,9 @@ export default function Marker({ property, id, isVisible, geometry, layer, featu
   const coordinates = useMemo(
     () =>
       geometry?.type === "Point"
-        ? geometry.coordinates
+        ? property?.height
+          ? [...geometry.coordinates, property.height]
```

Comment:
We should do like this, because`geometry.coordinates` might have height in the index of 3.
```suggestion
          ? (property.height ? [...geometry.coordinates.slice(0, 2), property.height] : geometry.coordinates)
```

Reply:
Done.